cmd/initializer_v2/dataset/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-alpine
```

Comment:
```suggestion
FROM python:3.11-slim-bookworm
```

Could you use the Debian image since the Alpine has a performance penalty due to without musl libc?
Python still depends on the C codes.

Reply:
Sure, let me create an issue